apps/nuxt3-ssr/components/table/Content.vue

```diff
@@ -26,22 +27,20 @@ let orderby = {
   [orderByColumn.value]: "ASC",
 };
 
-const rows = ref([]);
-const count = ref(0);
-
+let rows = ref([]);
+let count = ref(0);
 async function fetchRows() {
   const resp = await fetchGql(props.query, {
     ...props.filter,
     limit: pageSize,
     offset: offset.value,
     orderby,
   }).catch((e) => console.log(e));
-
-  rows.value = resp.data[props.type]?.map(props.rowMapper);
   count.value = resp.data[`${props.type}_agg`].count;
+  rows.value = resp.data[props.type]?.map(props.rowMapper);
 }
 
-fetchRows();
+await fetchRows();
```

Comment:
if used in a page ( or multiple on page ) the whole thing is going to be blocking , the use of the component has no real way of knowing this

Reply:
I had loading issues without the await. You believe it works without it?